interpreter/Makefile

```diff
@@ -9,131 +9,39 @@
 
 # Configuration
 
-NAME =		wasm
```

Comment:
Can we keep the NAME variable? That makes it easier to reuse the Makefile for derivatives.

Reply:
Fixed.